lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -248,6 +248,35 @@ function dosomething_northstar_save_id_field($uid, $northstar_response) {
 }
 
 /**
+ * Save the user's Northstar ID to their local profile.
+ *
+ * @param object $account
+ *   Fully loaded user object.
+ *
+ * @return string
+ *   The Northstar id or empty string on error.
+ */
+function dosomething_northstar_get_id($account = NULL) {
```

Comment:
It won't work as a handler for `dosomething_user_get_field()` because it'd fail on systems where Northstar is turned off (staging, local) because of this: https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1261.